setup.py

```diff
@@ -17,4 +17,14 @@
 # !/usr/bin/env python
 from setuptools import setup
 
-setup()
+setup(
+    entry_points={
```

Comment:
Shouldn't these go in setup.cfg instead?

Reply:
Yep, we had issues with issue points working in our TOML but cfg seems to work fine. Updated.